common/src/main/java/com/hortonworks/registries/common/util/FileStorage.java

```diff
@@ -20,13 +20,13 @@
 import java.util.Map;
 
 /**
- * Interface abstracting the upload and download of files (like parser jars, split/join or custom processor jars or any other files) for IoTaS.
- * IoTaS will provide a default file system based implementation which can be swapped by another implementation using fileStorageConfiguration
- * property in the iotas.yaml
+ * Interface abstracting the upload and download of files (like parser jars, split/join or custom processor jars or any other files) for Streamline.
+ * Streamline will provide a default file system based implementation which can be swapped by another implementation using fileStorageConfiguration
+ * property in the streamline.yaml
  */
 public interface FileStorage {
 
-    String DEFAULT_DIR = "/tmp/iotas-files";
+    String DEFAULT_DIR = "/tmp/streamline-files";
```

Comment:
should be name registry or something not streamline

Reply:
Do no use either registry or streamline but it can be like `/tmp/uploaded-files` or some other valid name.